packages/applications-service-api/src/middleware/parseQueryParamProperties.js

```diff
@@ -0,0 +1,24 @@
+const { parseInteger } = require('../utils/parse');
```

Comment:
Sorry approved but realised we should probably name this in kebab case like the other files

Reply:
Applies to the corresponding test file too